code/game/objects/items/food/sandwichtoast.dm

```diff
@@ -259,23 +259,47 @@
 	. = ..()
 	obj_flags &= ~UNIQUE_RENAME // You shouldn't be able to disguise this on account of how it kills you
 
-///Override for checkliked callback
+// Makes you feel disgusted if you look at it wrong.
+/obj/item/food/sandwich/death/examine(mob/user)
+	. = ..()
+	if(check_liked(user) == FOOD_LIKED)
+		return
+	// Only human mobs, not animals or silicons, can be disgusted by this.
+	if(!istype(user, /mob/living/carbon/human))
+		return
+	balloon_alert(user, "looked at it wrong!")
+	to_chat(user, span_warning("You imagine yourself eating [src]. You feel a sudden sour taste in your mouth, and a horrible feeling that you've done something wrong."))
```

Comment:
We don't really need both a balloon alert and a chat message, I would recommend just the chat message.